swig/hotrod_arch.i

```diff
@@ -71,7 +71,7 @@
     }
     %}
 
-%typemap(csinterfaces) infinispan::hotrod::ConnectionPoolConfigurationBuilder "IDisposable, Infinispan.HotRod.SWIG.ConnectionPoolConfigurationBuilder"
+%typemap(csinterfaces_derived) infinispan::hotrod::ConnectionPoolConfigurationBuilder "IDisposable, Infinispan.HotRod.SWIG.ConnectionPoolConfigurationBuilder"
```

Comment:
The ConnectionPoolConfigurationBuilder still looks like it does NOT extend AbstractConfigurationChildBuilder